Modules/Filtering/ImageStatistics/include/itkLabelOverlapMeasuresImageFilter.h

```diff
@@ -27,7 +27,7 @@ namespace itk
 {
 
 /** \class LabelOverlapMeasuresImageFilter
- * \brief Computes overlap measures between the set same set of labels of
+ * \brief Computes overlap measures between the same set of labels of
  * pixels of two images. Background is assumed to be 0.
```

Comment:
Thanks for fixing the typo @dzenanz Honestly I still don't understand it exactly... those two images have the "same set of labels", right? Are the "labels of pixels" just their pixel values? Anyway, no problem! Approved!

Reply:
> Are the "labels of pixels" just their pixel values?

Yes. I guess this is here to emphasize it does not do binary comparison, but label-wise comparison.